src/components/common/CityCard/cityCard.scss

```diff
@@ -1,9 +1,23 @@
-.card-wrapper{
+.card-wrapper {
     display: flex;
     flex-direction: column;
     height: 19.81rem;
     width: 11.25rem;
     background-color: rgba(255, 255, 255, 0.3);
```

Comment:
I thought that colors like white and black didn't require variables.

Reply:
Can I use rgba(white, 0.3) instead of variables?

Reply:
If you want a consistent experience across the whole app, you should have variables for black and white as well. Here is the thing, if the designer comes to you and asks to update all "black" colors to make it not so black, it will be easier to update variables in one place rather than looking for exact places and changing them.